recipes/nanorange/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
+#set_property( TARGET example PROPERTY CXX_STANDARD 17)
```

Comment:
Remove these comments

Reply:
I need another way of saying that this library must be compiled with -s compiler.cppstd=(>= c++17)

Reply:
please, take a look: https://github.com/conan-io/conan-center-index/issues/54
anyway, if your package requires c++17, uncomment this line to force the compiler compatibility.